rpm/sdk-user-manual.spec

```diff
@@ -35,7 +35,7 @@ for %{name}.
 
 %build
 %qmake5 -r VERSION=%{version}
-make
+make %{_smp_mflags}
```

Comment:
I know it wasn't your purpose, but this makes sdk-user-manual a quite nice test case/benchmark for the docker build engine :)